actioncable/lib/action_cable/server/broadcasting.rb

```diff
@@ -20,7 +20,10 @@ module Server
     module Broadcasting
       # Broadcast a hash directly to a named <tt>broadcasting</tt>. This will later be JSON encoded.
       def broadcast(broadcasting, message, coder: ActiveSupport::JSON)
-        broadcaster_for(broadcasting, coder: coder).broadcast(message)
+        payload = { broadcasting: broadcasting, message: message, coder: coder }
+        ActiveSupport::Notifications.instrument("broadcast.action_cable", payload) do
```

Comment:
I considered putting it there. The reason I put it in the broadcasting method on the Broadcasting module is it is the entry to the public API. The Broadcaster is an implementation detail. I would be happy to move it to the Broadcaster class if that is the best place.